### PR TITLE
Likvido.App/#20919 Adding new security service to maintain general we…

### PR DESCRIPTION
**Pull request by tomasgrilo**
Likvido/Likvido.App#20919

### CODE REVIEW

File: src/Likvido.Web/Services/Security/IWebSecurityService.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Likvido.Web.Services.IP;
+
+[PublicAPI]
+public interface IWebSecurityService
```
Comment (spewu) [inline src/Likvido.Web/Services/Security/IWebSecurityService.cs:7]: I think the name of this service is a bit too generic.. maybe we can limit the scope by naming it something like `IRedirectSecurityService` or something like that?

**Review by spewu — CHANGES_REQUESTED** [review]

File: src/Likvido.Web/Services/Security/IWebSecurityService.cs (hunk 1 above)
Comment (tomasgrilo) [reply src/Likvido.Web/Services/Security/IWebSecurityService.cs]: Agree, changed it

Comment (tomasgrilo) [reply src/Likvido.Web/Services/Security/IWebSecurityService.cs]: Got the commit names all wrong lol. Still refreshing this from vacation

**Review by spewu — CHANGES_REQUESTED** [review]
Super - thank you!

You need to do one more small thing, and that is increasing the version number in the /src/version.props file, as that will trigger the build a publishing of the package to NuGet :)

**Comment by tomasgrilo** [comment]
@spewu Done!

**Review by spewu — APPROVED** [review]

**Pull request merged**